pythonfile/flask_api.py

```diff
@@ -36,29 +39,103 @@ def add_project():
 
     if error == None:
         manifest['id']=uuid.uuid4()
+        #inserts in mongo db
         coll.insert(manifest)
-        elastic.store_json("test", "projects", manifest)
+        #inserts in elasticsearch now
+        res = es.index(index="projects-index", doc_type='Project', id=manifest['id'], body=manifest)
         return make_response(manifest['id'])
     else:
         return make_response((error, '302'))
 
-@app.route('/api/projects/<uuid:project_id>', methods=['GET'])
+@app.route('/api/projects/get/<project_id>')
 def get_project_by_id(project_id):
     res=coll.find_one({'_id':project_id})
     return jsonify(res)
 
-@app.route('/api/projects/<uuid:project_id>', methods=['DELETE'])
+
+@app.route('/api/projects/delete/<project_id>')
 def delete_project(project_id):
-    if coll.delete_one({'_id':project_id}).deleted_count != 0:
+    #deletion from elastic search index
+    try:
+        res = es.delete(index="projects-index", doc_type='Project', id=project_id, refresh=True)
+    except NotFoundError:
+        return make_response('Project not found', 404)
+    except:
+        return make_response('Unexpected Error', 500)
+    #Deletion from Mongodb and return
+    if (coll.delete_one({'_id':project_id}).deleted_count != 0):
         return make_response('Success')
     else:
         return make_response('Project not found', 404)
 
-# receive body of elasticsearch query
+
+
+#Note, works but need better exception handlin on empty index
 @app.route('/api/projects/search', methods=['POST'])
+#@app.route('/api/projects/search')
 def search():
-    res=es.search(index="test", doc_type="projects", body=request.json)
-    return jsonify(res)
+    try:
+        res=es.search(index="projects-index", doc_type="Project", body=request.json)
+        return jsonify(res)
```

Comment:
Could you add a TODO here?
I feel like the search result format is not very nice and we probably should filter it somehow or at least discuss this with the frontend team.